cmds/hostname/hostname.go

```diff
@@ -13,17 +13,15 @@ package main
 
 import (
 	"fmt"
-	"io"
+	"log"
 	"os"
 )
 
-func hostname(w io.Writer) error {
-	hostname, error := os.Hostname()
-	fmt.Fprintf(w, "%v", hostname)
-	return error
-}
-
 func main() {
-	hostname(os.Stdout)
-	fmt.Println()
+	hostname, err := os.Hostname()
```

Comment:
This was probably done this way for testing, but there's no solid way to test hostname, I guess.